src/Screens/Exchange/index.tsx

```diff
@@ -0,0 +1,212 @@
+// Copyright 2022-2023 @Polkasafe/polkaSafe-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+/* eslint-disable sort-keys */
+
+import { MinusCircleOutlined,PlusCircleOutlined } from '@ant-design/icons';
+import { OnrampWebSDK } from '@onramp.money/onramp-web-sdk';
+import { Dropdown, Form, Input } from 'antd';
+import { ItemType } from 'antd/es/menu/hooks/useItems';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import acalaLogo from 'src/assets/parachains-logos/acala-logo.png';
+import astarLogo from 'src/assets/parachains-logos/astar-logo.png';
+import kusamaLogo from 'src/assets/parachains-logos/kusama-logo.gif';
+import moonbeamLogo from 'src/assets/parachains-logos/moonbeam-logo.png';
+import moonriverLogo from 'src/assets/parachains-logos/moonriver-logo.png';
+import polkadotLogo from 'src/assets/parachains-logos/polkadot-logo.jpg';
+import polymeshLogo from 'src/assets/parachains-logos/polymesh-logo.png';
+import { ParachainIcon } from 'src/components/NetworksDropdown';
+import { useGlobalApiContext } from 'src/context/ApiContext';
+import { useGlobalUserDetailsContext } from 'src/context/UserDetailsContext';
+import { ONRAMP_APP_ID } from 'src/global/onrampAppId';
+import AddressInput from 'src/ui-components/AddressInput';
+import { CircleArrowDownIcon, ExternalLinkIcon } from 'src/ui-components/CustomIcons';
+import PrimaryButton from 'src/ui-components/PrimaryButton';
+import getEncodedAddress from 'src/utils/getEncodedAddress';
+import styled from 'styled-components';
+
+export const onrampTokens = {
+	POLKADOT: 'polkadot',
+	KUSAMA: 'kusama',
+	ASTAR: 'astar',
+	MOONBEAM: 'moonbeam',
+	MOONRIVER: 'moonriver',
+	POLYMESH: 'polymesh',
+	ACALA: 'acala'
+};
+
+export const onrampTokenProperties = {
```

Comment:
this data should be in network constants, both `onrampTokens` and `onrampTokenProperties`